tables/guardduty_finding_log/guardduty_finding.go

```diff
@@ -0,0 +1,141 @@
+package guardduty_finding_log
```

Comment:
```suggestion
package guardduty_finding
```

For the package name + dir name, I think we should remove `_log` since it's just `guardduty_finding`.